lib/rspec/rails/view_rendering.rb

```diff
@@ -102,7 +102,7 @@ def find_templates(*args)
 
       # @private
       class EmptyTemplateHandler
-        def self.call(_template)
+        def self.call(*)
```

Comment:
It's preferable to have 1 required, 1 optional, just because it matches their API